cairo_programs/fibonacci.cairo

```diff
@@ -0,0 +1,18 @@
+func main():
+    # Call fib(1, 1, 10).
+    let result: felt = fib(1, 1, 10)
+
+    # Make sure the 10th Fibonacci number is 144.
+    assert result = 144
+    ret
```

Comment:
It's better to use return() instead as it's less low level.

Reply:
I just used the program we already have in `cairo-rs`, it was really to check for consistency :D